<launch>
	<param name="calibration" value="/home/yangjl/Desktop/sensor/VLP-16.yaml"/>
	<param name="max_range" value="130.0"/>
	<param name="min_range" value="0.3"/>	
	<param name="device_ip" value="192.168.1.200"/>
	<param name="frame_id" value="velodyne"/>
	<param name="model" value="VLP16"/>
	<param name="pcap" value=""/>
	<param name="port" value="2368"/>
	<param name="read_fast" value="false"/>
	<param name="read_once" value="false"/>
	<param name="repeat_delay" value="0.0"/>
	<param name="rpm" value="600.0"/>
	<param name="cut_angle" value="-0.01"/>
	<param name="ring" value="-1"/>
	<param name="resolution" value="0.007"/>
	<node pkg="velodyne" name="velodyne_pointcloud" type="velodyne_pointcloud" output="screen"/>
	<node pkg="velodyne" name="velodyne_driver" type="velodyne_driver" output="screen"/>
</launch>
